### DISCUSSION TITLE
VSIXCookbook Walkthrough to Create Language Editor

### DISCUSSION THREAD

**Discussion by ghost** [VsixCommunity/Community.VisualStudio.Toolkit#264] [Ideas]
@madskristensen; Based on Hacking Visual Studio session on 1/14/2022, you talked about the language editor. I am working on a free project that would allow you to edit and test (.ling) extension ([LinqPad v7](https://www.linqpad.net/)) queries inside of Visual Studio 2022 or even just LINQ queries inside your current CSharp project files. I tried based on you Markdown Editor V2 to create this editor but was not successful since the parser was from a third party, but after your demo of [PkgdefLanguage](https://github.com/madskristensen/PkgdefLanguage) and its language editor with its own parser, I would like to try again. If it is all right with you, I would like to create a Walkthrough for this attempt and add it to the [Visual Studio Extensibility Cookbook](https://www.vsixcookbook.com/) site which would include a markdown ReadMe file and source code link to the GitHub project. I have contributed to the [vsixcookbook ](https://www.vsixcookbook.com/)site before and like it as a sole source Goto site for info on creating Visual Studio Extensions. 
(@SFC-Sarge Danny McNaught)

**Comment by madskristensen** [comment]
That would be amazing

**Comment by ghost** [comment]
Working on it now, Thanks.

Danny McNaught
Computer Question

email: ***@***.***

________________________________
